### ISSUE TITLE
Pressing Z doesnt stash any items, 

### ISSUE THREAD

**Issue by Andryj98** [ImJustMatt/ExpandedStorage#96]
Pressing Z doesnt stash any items, tried to configure the settings nothing changed, no error message or warning is shown

**Comment by bambanah** [comment]
Have you enabled "Categorize Chest"?

**Comment by Andryj98** [comment]
Yes, i figured few other things, it is on multiplayer, i left all default
settings and it appears that better crafting or maybe even ui info suite 2
overrides or somehow blocks proper interface to show the interface when you
open the chest because the logos at the top do not appear, after i pick an
item from the chest or place anything into it the the top part appears and
it usually lets to open the chest while pressing B and store items (most of
the times once) with Z, also for other player when opening crafting menu
appears empty with 2.18.1 version if i remember correctly

**Comment by bambanah** [comment]
I'm getting this now too! Seems to happen for me when playing multiplayer, but weirdly only from the second day of the session?

It seems to work again when we go to title and then back in. Very strange.

**Comment by JButgereit** [comment]
Same issue, stash to chests just stops working after a day or so in game.

 [Full log here.](https://smapi.io/log/96f930922fe645f0a16b55b278f11c7d)

**Comment by splintermind** [comment]
Same issue, stash to chests just stops working after a day or so in game.

 [Full log here.](https://smapi.io/log/96f930922fe645f0a16b55b278f11c7d)
